mlocal/frags/Makefile.stub

```diff
@@ -2,23 +2,25 @@
 # like go_TAG / WRAPPER_LIBS
 include Makefile.cond
 
+EXEC_PREFIX := $(PREFIX)
+
 libruntime := $(BUILDDIR)/lib/libruntime.a
 
 starter_OBJ := $(shell $(SOURCEDIR)/makeit/gengodep $(SOURCEDIR)/src/runtime/starter/smaster.go)
 starter_OBJ += $(shell $(SOURCEDIR)/makeit/gengodep $(SOURCEDIR)/src/runtime/starter/scontainer.go)
 starter_OBJ += $(shell $(SOURCEDIR)/makeit/gengodep $(SOURCEDIR)/src/runtime/starter/rpc.go)
 
 starter := $(BUILDDIR)/src/runtime/starter/c/starter
-starter_INSTALL := $(PREFIX)/libexec/singularity/bin/starter
-starter_suid_INSTALL := $(PREFIX)/libexec/singularity/bin/starter-suid
+starter_INSTALL := $(EXEC_PREFIX)/libexec/singularity/bin/starter
+starter_suid_INSTALL := $(EXEC_PREFIX)/libexec/singularity/bin/starter-suid
 
 
 singularity := $(BUILDDIR)/singularity
-singularity_INSTALL := $(PREFIX)/bin/singularity
+singularity_INSTALL := $(EXEC_PREFIX)/bin/singularity
```

Comment:
We're _wanting_ to get all the base autoconf options added in. So...  This _should_ end up being:
`$(BINDIR)/singularity`

Where BINDIR defaults to `$(EXEC_PREFIX)/bin` ... and `$(EXEC_PREFIX)` defaults to `$(PREFIX)`

In general, most of these options:
https://www.gnu.org/prep/standards/html_node/Directory-Variables.html

The idea is to pass to `mconfig` in the `%build` section of the spec like: 
`./mconfig -p %{_prefix} -e %{_exec_prefix} -l %{_libexecdir} -D %{_datadir}`
etc, etc... 

So we are using site specific RPM macro's when building.

When using autoconf, it was done automagically.

Reply:
Ok, but is this good enough as an intermediate step?

Reply:
Yes. This was more of a general comment on the wanted makeit usage, nothing against how it was implemented there.